packages/form-builder/src/settings/donation-goal/index.js

```diff
@@ -79,6 +71,19 @@ const DonationGoalSettings = () => {
                             onChange={(goalFormat) => dispatch(setFormSettings({goalFormat}))}
                         />
                     </PanelRow>
+                    <PanelRow>
+                        {/*
+                            @todo Do we need to reset the goal amount when the goal format changes?
+                                The context can be wildly different, ie Amount Raised vs Number of Donors.
+                                Perhaps we could implement smart default amounts for each goal format?
```

Comment:
Good thoughts, we should discuss this further.  I think some simple conditional logic will be a fine start for the `goalAmount` and description based on the goal format